plugins/rollbar_dashboards/app/views/samson_rollbar_dashboards/_fields.html.erb

```diff
@@ -19,5 +20,11 @@
     <div class="col-lg-3">
       <%= setting.text_field :read_token, class: "form-control", placeholder: "Project Read Token" %>
     </div>
+    <div class="col-lg-3">
+      <%= setting.text_field :account_and_project_name,
+            class: "form-control",
+            placeholder: "Account Name/Project Name"
```

Comment:
kinda looks like "put this or that here" ... maybe `{Account Name}/{Project Name}` to be more explicit